dockerfile-image-update/src/test/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtilTest.java

```diff
@@ -488,7 +548,7 @@ public void testCreatePullReq_Delete() throws Exception {
         when(gitHubUtil.createPullReq(any(), anyString(), any(), anyString(), eq(Constants.PULL_REQ_ID))).thenReturn(1);
         doCallRealMethod().when(gitHubUtil).safeDeleteRepo(forkRepo);
         dockerfileGitHubUtil = new DockerfileGitHubUtil(gitHubUtil);
-        dockerfileGitHubUtil.createPullReq(new GHRepository(), "branch", forkRepo, "Automatic Dockerfile Image Updater");
+        dockerfileGitHubUtil.createPullReq(new GHRepository(), "branch", forkRepo, null);
```

Comment:
why set to null?

Reply:
Just to test the null scenario. We have a test that already provides a non-message.